runtime/runtime.go

```diff
@@ -632,9 +643,13 @@ func (r *interpreterRuntime) newInterpreter(
 			r.injectedCompositeFieldsHandler(runtimeInterface, runtimeStorage),
 		),
 		interpreter.WithUUIDHandler(func() (uuid uint64) {
+			var err error
```

Comment:
Should this error be handled here or on the location where it is called?

Reply:
@janezpodhostnik Do you mean `Interpreter.UUIDHandler` should be have an error return value and this handler should just pass it down?

Reply:
Yep, not sure which is better here? Passing it down or handling it here.

Reply:
Refactored it in d37c9f3